src/python/turicreate/toolkits/drawing_classifier/_tf_drawing_classifier.py

```diff
@@ -27,6 +27,12 @@ def __init__(self, net_params, batch_size, num_classes):
         for key in net_params.keys():
             net_params[key] = _utils.convert_shared_float_array_to_numpy(net_params[key])
 
+        if 'random_seed' in config:
+            seed = int(_utils.convert_shared_float_array_to_numpy(config['random_seed']))
+            _tf.set_random_seed(seed)
+        else:
+            seed=None
```

Comment:
Let's follow PEP8 and add spaces around the assignment.

"Always surround these binary operators with a single space on either side: assignment (=), augmented assignment (+=, -= etc.), comparisons (==, <, >, !=, <>, <=, >=, in, not in, is, is not), Booleans (and, or, not)." from https://www.python.org/dev/peps/pep-0008/

Reply:
I think I can remove the else completely. It is not needed to assign `seed` to None.